packages/MarkdownEditor-Core.package/MarkdownLine.class/instance/potentialBlockClass.st

```diff
@@ -0,0 +1,6 @@
+accessing
+potentialBlockClass
```

Comment:
I don't like the name. When I first read it, I had no idea what it meant. Maybe `potentialMatchingBlockClass`. This expresses that it tries to find a matching class. But maybe it is a little bit to fastidious (I don't have a clue what's the right word for penibel: https://www.dict.cc/?s=penibel 😄)

Reply:
So my thought process was that the line is potentially converted into a block of the potential block class. You are right that we want the class which matches the line. But we have renamed `Block>>matches:` to `mayBeginWith:` to be consistent with the tokens. I think it would be weird to now bring the word match back into the name because we don't use it like that. I would also be okay with just calling it `blockClass` since the matching (which we don't call matching..) is definite and not really potential. Would that be clearer to you?

Reply:
Yep, that would help.